site/src/atomics/typography.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Typography Atomics
+description: Typography related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Typography Atomics
+
+The typography scale is designed to for great readability across the platform. This page covers the various classes available to modify text.
+
+## Font size
+
+There are two avenues for modifying font sizes in Atlas.
+
+#### Heading sizes
+
+The following classes can be used to accurately adopt a heading's size (h1- h6).
```

Comment:
suggested addition: 

These sizes are tied to the default heading sizes render within articles and the [markdown component](../components/markdown.md).

Reply:
Added